generators/spring-controller/index.js

```diff
@@ -34,6 +34,13 @@ module.exports = class extends BaseGenerator {
         this.argument('name', { type: String, required: true });
         this.name = this.options.name;
 
+        this.option('default', {
+            type: String,
```

Comment:
Shouldn't this be type boolean?

Reply:
indeed! thanks Deepu, I will update this, later